plugins/http_plugin/http_plugin.cpp

```diff
@@ -463,8 +463,9 @@ namespace eosio {
                   return;
                }
 
-               // post to the app thread capturing the body_scope
-               app().post( priority, [next_ptr, conn=std::move(conn), r=std::move(r), tracked_b=std::move(tracked_b), then=std::move(then)]() mutable {
+               // post to the app thread taking shared ownership of next (via std::shared_ptr),
+               // sole ownership of the tracked body and the passed in parameters
+               app().post( priority, [next_ptr=std::move(next_ptr), conn=std::move(conn), r=std::move(r), tracked_b=std::move(tracked_b), then=std::move(then)]() mutable {
```

Comment:
This one can't be a move. The other one could be, but not this one.

Reply:
🤦‍♂